ofrak_core/ofrak/cli/ofrak_cli.py

```diff
@@ -127,6 +127,11 @@ def add_ofrak_arguments(command_subparser):
             choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
             default=OFRAK.DEFAULT_LOG_LEVEL,
```

Comment:
If the `DEFAULT_LOG_LEVEL` is a field of the `OFRAK` class, does it make sense for the `DEFAULT_LOG_FILE` path to be as well?

Reply:
No opinion here - I kept it at top level, where it was, but could be moved into `OFRAK` class, I guess.

Reply:
I think it's fine to leave as-is. I'll resolve this conversation and run the tests. Once they pass, this should be good to merge